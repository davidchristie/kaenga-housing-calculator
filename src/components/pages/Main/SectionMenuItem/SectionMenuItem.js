import React, { Component, PropTypes } from 'react'
import { ListGroupItem } from 'react-bootstrap'

class SectionMenuItem extends Component {
  render () {
    return (
      <ListGroupItem>
        {this.props.children}
      </ListGroupItem>
    )
  }
}

SectionMenuItem.propTypes = {
  to: PropTypes.string
}

export default SectionMenuItem
