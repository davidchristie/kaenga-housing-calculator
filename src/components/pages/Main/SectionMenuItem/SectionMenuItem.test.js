import React from 'react'
import ReactTestUtils from 'react-dom/test-utils'

import SectionMenuItem from './SectionMenuItem'

const shallowRenderer = ReactTestUtils.createRenderer()

it('matches snapshot', () => {
  expect(shallowRenderer.render(<SectionMenuItem />)).toMatchSnapshot()
})
