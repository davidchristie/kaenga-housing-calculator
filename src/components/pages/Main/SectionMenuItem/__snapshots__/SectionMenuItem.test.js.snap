exports[`test matches snapshot 1`] = `
<ListGroupItem
  bsClass="list-group-item"
  href={undefined}
  listItem={false} />
`;
